packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -25,8 +25,8 @@ const PageWrapper = styled.div`
     overflow-x: hidden;
 `;
 
-const Body = styled.div`
-    position: relative;
+const Body = styled.div<{ elsOpen?: boolean }>`
+    position: ${props => (props.elsOpen ? `unset` : `relative`)};
```

Comment:
Maybe it causes some problems
![Screenshot 2021-11-02 at 13 32 41](https://user-images.githubusercontent.com/33235762/139847806-da4cdf74-e7ed-476d-971a-5d1a85e1afb2.png)

Reply:
c084b62